### ISSUE TITLE
Update requirements

### ISSUE THREAD

**Issue by Danielss89** [ZF-Commons/ZfcUser#474]
Current release needs ZF2.3 but in composer.json it says ~2.1

**Comment by adamlundrigan** [comment]
If that's the case, I agree with @stijnhau. The `php` line could also be dropped as ZF2 defines it's own minimum PHP version, so this line is redundant.

I usually set up my projects to test against the latest revision of each minor ([example](https://github.com/adamlundrigan/LdcUserProfile/blob/master/.travis.yml#L10)).  I can PR a similar .travis.yml to test multiple ZF2 versions if you like.

**Comment by adamlundrigan** [comment]
Quick scan of minimum supported ZF2 versions for ZfcUser:

               | ZfcUser 2.x | ZfcUser 1.x |
------------ | --------------- | --------------- |
ZF 2.0.x | N | ? |
ZF 2.1.x | N | Y |
ZF 2.2.x | N | Y |
ZF 2.3.x | Y | Y |
ZF dev-master | Y | Y |

ZfcUser 1.x tests use `Zend\Session\AbstractContainer`, which wasn't available in 2.0.x.

ZfcUser 2.x `ZfcUser\Service\User` hints on `Zend\Authentication\AuthenticationServiceInterface`, which was added in [ZF 2.3.0](https://github.com/zendframework/zf2/pull/5901)

Script used:
```
export ZF2_VER="dev-master@dev"; sed -i "s/\"\(zendframework\/zend-[^\\\"]\+\)\".*\([,?]\)$/\"\\1\": \"$ZF2_VER\"\\2/g" composer.json ; vim comcomposer update --dev && ./vendor/bin/phpunit -c `pwd`/tests/phpunit.xml
```
(Except for `2.0.*@stable` and `dev-*@dev`, where I changed composer.json to point to `zendframework/zendframework` instead of individual packages)

**Comment by stijnhau** [comment]
Oke is the table of @adamlundrigan is correct the PR is indead only needed for the 2.0.0
So @Danielss89 when do you create the PR?
